ts/src/binance.ts

```diff
@@ -2343,15 +2343,15 @@ export default class binance extends Exchange {
         });
     }
 
-    isInverse (type, subType = undefined): boolean {
+    isInverse (type: string, subType: SubType = undefined): boolean {
```

Comment:
subType needs to be imported in python, just use Str here

Reply:
I just added the type `SubType` in python, but I can still change it to Str if you would like

Reply:
yes I would rather change it to str